react-native-libraries.json

```
@@ -9881,10 +9881,12 @@
   },
   {
     "githubUrl": "https://github.com/AlirezaHadjar/expo-drag-drop-content-view",
```

```suggestion
    "githubUrl": "https://github.com/AlirezaHadjar/expo-drag-drop-content-view/tree/main/packages/expo-drag-drop-content-view",
```

The package URL also need to be updated, to correctly extract the package data after monorepo reorganization.

---

Correct, thanks for the tip🙂